packages/suite/src/actions/wallet/signVerifyActions.ts

```diff
@@ -1,133 +1,130 @@
-
-export const sign = (path: [number], message: string, hex = false) => async (
-    dispatch: Dispatch,
-    getState: GetState,
-) => {
-    const selectedDevice = getState().suite.device;
-    if (!selectedDevice) return;
-
-    const response = await TrezorConnect.ethereumSignMessage({
-        device: {
-            path: selectedDevice.path,
-            instance: selectedDevice.instance,
-            state: selectedDevice.state,
-        },
-        path,
-        hex,
-        message,
-        useEmptyPassphrase: selectedDevice.useEmptyPassphrase,
-    });
-
-    if (response && response.success) {
-        dispatch({
-            type: SIGN_VERIFY.SIGN_SUCCESS,
-            signSignature: response.payload.signature,
-        });
-    } else {
+    if (typeof errorResponse === 'string') {
         dispatch(
             notificationActions.addToast({
-                type: 'sign-message-error',
-                error: response.payload.error,
+                type: 'error',
+                error: errorResponse,
             }),
         );
+    } else {
+        dispatch(notificationActions.addToast({ type: 'copy-to-clipboard' }));
     }
 };
 
-export const verify = (address: string, message: string, signature: string, hex = false) => async (
+export const sign = (message: string, path: string, hex = false) => async (
     dispatch: Dispatch,
     getState: GetState,
-) => {
-    const selectedDevice = getState().suite.device;
-    if (!selectedDevice) return;
-    const error = validateAddress(address);
+): Promise<string | boolean> => {
+    const { device } = getState().suite;
+    const { account } = getState().wallet.selectedAccount;
+    if (!device || !account) return false;
 
-    if (error) {
-        dispatch({
-            type: SIGN_VERIFY.ERROR,
-            inputName: 'verifyAddress',
-            message: error,
-        });
+    let fn;
+
+    switch (account.networkType) {
+        case 'bitcoin': {
+            fn = TrezorConnect.signMessage;
+            break;
+        }
+        case 'ethereum': {
+            fn = TrezorConnect.ethereumSignMessage;
+            break;
+        }
+        default: {
+            fn = () => ({
+                success: false,
+                payload: {
+                    error: `Unsupported network: ${account.networkType}`,
+                    code: undefined,
+                    signature: '',
+                },
+            });
+            break;
+        }
     }
 
-    if (!error) {
-        const response = await TrezorConnect.ethereumVerifyMessage({
-            device: {
-                path: selectedDevice.path,
-                instance: selectedDevice.instance,
-                state: selectedDevice.state,
-            },
-            address,
-            message,
-            signature,
-            hex,
-            useEmptyPassphrase: selectedDevice.useEmptyPassphrase,
-        });
+    const params = {
```

Comment:
add `{
  device,
  useEmptyPassphrase: device.useEmptyPassphrase,
}` to params

Reply:
still missing

Reply:
this was not described in the API, psychics on vacation, but i rewrote this